scripts/celo/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env sh
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+# Exit on failure
+set -ex
+
+rm -rf /root/celo
+geth init --datadir /root /root/genesis.json
+
+geth --datadir /root --maxpeers 0 --light.maxpeers 0 --password /root/password.txt account import /root/key.txt
+#rm -rf /root/keystore
+#cp -r /root/keystore /root/.ethereum/
```

Comment:
You need to be importing the 5 default keys we use. If you add the correct flags below, it will automatically import/unlock them, provided you run these last two commands